zowe/zowe-cli-standalone-package
# Fix Zowe CLI Coverage workflow

**What It Does**

Updates names of CLI coverage artifacts

**How to Test**

See successful workflow run: https://github.com/zowe/zowe-cli-standalone-package/actions/runs/11664470951

**Review Checklist**
I certify that I have:


**Additional Comments**
